apps/accounts/api/automations/change_secret_dashboard.py

```diff
@@ -121,12 +123,12 @@ def get(self, request, *args, **kwargs):
 
         if _all or query_params.get('total_count_change_secrets'):
             data['total_count_change_secrets'] = self.get_filtered_counts(
-                self.change_secrets_queryset, 'date_updated'
+                self.change_secrets_queryset
             )
 
         if _all or query_params.get('total_count_periodic_change_secrets'):
             data['total_count_periodic_change_secrets'] = self.get_filtered_counts(
-                self.change_secrets_queryset.filter(is_periodic=True), 'date_updated'
+                self.change_secrets_queryset.filter(is_periodic=True)
             )
 
         if _all or query_params.get('total_count_change_secret_assets'):
```

Comment:
I didn't find any significant differences between the codes provided. Both show the same functionality but with slight syntactic changes to maintain readability and avoid confusion of variable names or operations that may have been used at different points in time.